huaweicloud/resource_huaweicloud_rds_instance_v3.go

```diff
@@ -502,12 +499,13 @@ func resourceRdsInstanceV3Update(d *schema.ResourceData, meta interface{}) error
 	// Since the instance will throw an exception when making an API interface call in 'BACKING UP' state,
 	// wait for the instance state to be updated to 'ACTIVE' before calling the interface.
 	stateConf := &resource.StateChangeConf{
-		Pending:    []string{"BACKING UP", "ACTIVE"},
-		Target:     []string{"ACTIVE"},
-		Refresh:    rdsInstanceStateRefreshFunc(client, instanceID),
-		Timeout:    d.Timeout(schema.TimeoutDefault),
-		Delay:      10 * time.Second,
-		MinTimeout: 3 * time.Second,
+		Pending:      []string{"BACKING UP"},
+		Target:       []string{"ACTIVE"},
+		Refresh:      rdsInstanceStateRefreshFunc(client, instanceID),
+		Timeout:      d.Timeout(schema.TimeoutDefault),
+		PollInterval: 5 * time.Second,
```

Comment:
the **Delay** field is missing

Reply:
remove pending array and add delay with 5 second.